Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/buttons/Button.tsx

```diff
@@ -96,14 +99,22 @@ const Button = ({
         ? styles.wide
         : '';
 
-    const buttonClassName = concatClassNames(
-        styles.btn,
-        typeClassName,
-        sizeClassName,
-        stateClassName,
-        wideClassName,
-        className,
-    );
+    const buttonClassName = matchOnlyInternalClass
```

Comment:
No. Internal class is to be managed internally not by a flag passed to the component.